frontend/src/main/components/OurTable.js

```diff
@@ -42,7 +42,9 @@ export default function OurTable({ columns, data, testid = "testid", ...rest })
                     <tr {...headerGroup.getHeaderGroupProps()}>
                         {headerGroup.headers.map(column => (
                             <th
-                                {...column.getHeaderProps(column.getSortByToggleProps())}
+                                {...column.getHeaderProps({
+                                    ...column.getSortByToggleProps(), // Include sorting functionality
```

Comment:
Same with the comma on this line, was this a leftover from an undone change?

Reply:
npx eslint --fix changed this I believe